### PR TITLE
Set up non-heroku deploy workflow

### PR DESCRIPTION
**Pull request by tkkuehn**
## Proposed changes

This PR adds a workflow to build a "release" consisting of a built wheel of the AFIDs validator and the a set of environment variables to configure it (drawn from the repository secrets), then deploy it to the production AFIDs Validator server (also configured via the repository secrets).

I've tested this from a branch on my fork and everything seems to be in order, but its worth noting that a couple of things outside of merging this PR will need to happen for the workflow to work:

- [x] The appropriate config variables will need to be added as secrets. (Done but will need to be verified)
- [x] I'll need to take a look at the production server to make sure it's set up to handle the architecture proposed here.
- [x] SSH keys will need to be generated and added to the repo as secrets and to the production server.
- [x] I'll need to configure the production server to accept SSH requests from any IP.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md)_

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        run: |
+          DATE=`date +%s`
+          mkdir -p releases/afidsvalidator-"$DATE"
+          cp .env releases/afidsvalidator-"$DATE"
+          WHEEL=`ls dist | grep whl`
+          cp dist/"$WHEEL" releases/afidsvalidator-"$DATE"
+          echo WHEEL="$WHEEL" >> $GITHUB_ENV
+          echo DATE="$DATE" >> $GITHUB_ENV
+
+      - name: Deploy release
```
Comment (kaitj) [inline .github/workflows/deploy.yml:63]: More of a consistency thing, but would you be opposed to having `env` before `run`?

File: fabrictasks.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+def push_release(
+    connection: Connection,
+    new_release_path: Path,
+    remote_releases_dir: PurePath,
+    wheel_name: str,
+):
+    remote_release_new = str(remote_releases_dir / new_release_path.name)
+    connection.run(f"mkdir {remote_release_new}")
```
Comment (kaitj) [inline fabrictasks.py:14]: Assuming this is running a bash / shell command, do we need to consider parent directories that need to be created?

**Review by kaitj — APPROVED** [review]
It looks good to me. One minor comment and a question.

File: fabrictasks.py (hunk 2 above)
Comment (tkkuehn) [reply fabrictasks.py:14]: Ah I think ideally we would set this directory structure ahead of time, but I agree that making this `f"mkdir -p {remote_release_new}"` could avoid issues.

**Comment by tkkuehn** [comment]
Note to self: Pretty much all the prep work is done, just need to update the service definition on the server before merging and trying this out.

**Pull request merged**